File: easy/gotham/gotham.rb
"""
@author	André Peralta
@version 20/05/2021
"""

def help? n_criminals 

	if n_criminals < 5
		return "I got this!"
	elsif n_criminals >= 5 and n_criminals <= 10
		return "Help me Batman"
	elsif n_criminals > 10
		return "Good Luck out there!"
	end

end


puts "Number of criminals: "
n_criminals = gets.to_i

print help?(n_criminals)

File: easy/gotham/gotham.c
#include <stdio.h>
#include <math.h>

void main() {

    int criminals;

    printf("Introduce el número de criminales: ");
    scanf("%d",&criminals);
    
    if(criminals < 5) printf("I got this!");
    else if(criminals >= 5 && criminals < 10)printf("Help me Batman");
    else if(criminals > 10) printf("Good Luck out there!");
}
